ts/Series/CompareCumulativeComposition.ts

```diff
@@ -0,0 +1,789 @@
+/* *
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
```

Comment:
What and how should the copyright and license information be placed her, @TorsteinHonsi?